src/sim65/6502.c

```diff
@@ -4,31 +4,13 @@
 /*                                                                           */
 /*                           CPU core for the 6502                           */
 /*                                                                           */
+/* This file is part of                                                      */
+/* cc65 - a freeware C compiler for 6502 based systems                       */
 /*                                                                           */
+/* https://github.com/cc65/cc65                                              */
 /*                                                                           */
-/* (C) 2003-2012, Ullrich von Bassewitz                                      */
-/*                Roemerstrasse 52                                           */
-/*                D-70794 Filderstadt                                        */
-/* EMail:         uz@cc65.org                                                */
-/*                                                                           */
-/* Mar-2017, Christian Krueger, added support for 65SC02                     */
-/*                                                                           */
-/* This software is provided 'as-is', without any expressed or implied       */
-/* warranty.  In no event will the authors be held liable for any damages    */
-/* arising from the use of this software.                                    */
-/*                                                                           */
-/* Permission is granted to anyone to use this software for any purpose,     */
-/* including commercial applications, and to alter it and redistribute it    */
-/* freely, subject to the following restrictions:                            */
-/*                                                                           */
-/* 1. The origin of this software must not be misrepresented; you must not   */
-/*    claim that you wrote the original software. If you use this software   */
-/*    in a product, an acknowledgment in the product documentation would be  */
-/*    appreciated but is not required.                                       */
-/* 2. Altered source versions must be plainly marked as such, and must not   */
-/*    be misrepresented as being the original software.                      */
-/* 3. This notice may not be removed or altered from any source              */
```

Comment:
IANAL, but does this change really qualify as "This notice may not be removed or altered from the source distribution"?

Yes, I see that the LICENSE file is there. But, from what I understood from Uz' intentions, he wanted to make sure that this notice is retained even if someone takes only one source file (and forgets to create a "whole distribution")

Unless someone more knowledgeable in the legal aspects of licenses confirms otherwise, I would not remove these notices from the individual files.

But YMMV.

Reply:
Good point. I've contacted Uz how he sees this issue.